prime-dotnet-webapi/Models/OboSite.cs

```diff
@@ -20,12 +20,20 @@ public class OboSite : BaseAuditable, IEnrolleeNavigationProperty
         [JsonIgnore]
         public CareSetting CareSetting { get; set; }
 
+        public HealthAuthorityCode? HealthAuthorityCode { get; set; }
+
+        [JsonIgnore]
+        public HealthAuthority HealthAuthority { get; set; }
+
         public string SiteName { get; set; }
 
         public string PEC { get; set; }
 
         public string FacilityName { get; set; }
 
+        [Required]
+        public string JobTitle { get; set; }
```

Comment:
If we are adding jobs to the oboSites, then we should probably remove the jobs Collection on Enrollee and migrate all jobs to exist within an oboSite

Reply:
I agree re: "remove the jobs Collection on Enrollee" and at some point there should be a Migration to drop the `Job` table.  

Related, the previous arrangement of associating jobs at the enrollee-level makes it difficult to know which Obo Site existing Jobs (in production) should be associated with, *if* an enrollee has more than one Care Setting/Health Authority.

Reply:
I think before this can be merged in we will have to come up with a solution for the migration. Agreed though it will be a tricky one, will need some discussion

Reply:
The proposed data migration is in the comments of the Jira ticket:  https://bcgovmoh.atlassian.net/browse/PRIME-1459